_sass/components/wells.scss

```diff
@@ -13,11 +13,11 @@
 }
 
 .well-anchor {
-  color: $gray-dark;
   cursor: pointer;
   display: flex;
   font-size: 21px;
   padding: 30px;
+  transition: 0.125s ease-in-out box-shadow;
```

Comment:
The property should always come first, the preferred syntax is: `transition: [property] [duration] [delay] [timing]`. Also no leading zeros.